src/Raven.Server/NotificationCenter/Notifications/Server/ClusterTopologyChanged.cs

```diff
@@ -53,7 +53,8 @@ public override DynamicJsonValue ToJson()
                 NodeTag = nodeTag,
                 Status = status,
                 CurrentTerm = term,
-                NodeLicenseDetails = nodeLicenseDetails
+                NodeLicenseDetails = nodeLicenseDetails,
+                IsPersistent = true
```

Comment:
Why do we want to persist this?

Reply:
1. We need to update the nodes topology after reelection even if they are connected after the initial message was sent.
2. Anyway we keep only the recent topology. (the Id defined by the Type in this case)